src/__tests__/shared/components/input/TestInputWithLabelFormGroup.test.tsx

```diff
@@ -8,92 +8,86 @@ describe('text input with label form group', () => {
   describe('layout', () => {
     it('should render a label', () => {
       const expectedName = 'test'
-      const wrapper = shallow(
+      const expectedLabel = 'label test'
+      render(
         <TextInputWithLabelFormGroup
           name={expectedName}
-          label="test"
+          label={expectedLabel}
           value=""
           isEditable
           onChange={jest.fn()}
         />,
       )
 
-      const label = wrapper.find(Label)
-      expect(label).toHaveLength(1)
-      expect(label.prop('htmlFor')).toEqual(`${expectedName}TextInput`)
-      expect(label.prop('text')).toEqual(expectedName)
+      expect(screen.getByText(expectedLabel)).toHaveAttribute('for', `${expectedName}TextInput`)
     })
 
     it('should render a text field', () => {
-      const expectedName = 'test'
-      const wrapper = shallow(
+      const expectedLabel = 'test'
+      render(
         <TextInputWithLabelFormGroup
-          name={expectedName}
-          label="test"
+          name="test"
+          label={expectedLabel}
           value=""
           isEditable
           onChange={jest.fn()}
         />,
       )
 
-      const input = wrapper.find(TextInput)
-      expect(input).toHaveLength(1)
+      expect(screen.getByLabelText(expectedLabel)).toBeInTheDocument()
     })
 
     it('should render disabled is isDisable disabled is true', () => {
-      const expectedName = 'test'
-      const wrapper = shallow(
+      const expectedLabel = 'test'
+      render(
         <TextInputWithLabelFormGroup
-          name={expectedName}
-          label="test"
+          name="test"
+          label={expectedLabel}
           value=""
           isEditable={false}
           onChange={jest.fn()}
         />,
       )
 
-      const input = wrapper.find(TextInput)
-      expect(input).toHaveLength(1)
-      expect(input.prop('disabled')).toBeTruthy()
+      expect(screen.getByLabelText(expectedLabel)).toBeDisabled()
     })
 
     it('should render the proper value', () => {
-      const expectedName = 'test'
+      const expectedLabel = 'test'
       const expectedValue = 'expected value'
-      const wrapper = shallow(
+      render(
         <TextInputWithLabelFormGroup
-          name={expectedName}
-          label="test"
+          name="test"
+          label={expectedLabel}
           value={expectedValue}
           isEditable={false}
           onChange={jest.fn()}
         />,
       )
 
-      const input = wrapper.find(TextInput)
-      expect(input).toHaveLength(1)
-      expect(input.prop('value')).toEqual(expectedValue)
+      expect(screen.getByLabelText(expectedLabel)).toHaveAttribute('value', expectedValue)
```

Comment:
Slightly simplified:

```suggestion
      expect(screen.getByLabelText(expectedLabel)).toHaveValue(expectedValue)
```

Reply:
Thanks for the feedback, I will update it right now :D